Dockerfile

```diff
@@ -0,0 +1,18 @@
+
+FROM openshift/origin-release:golang-1.10 as builder
+WORKDIR /go/src/github.com/operator-framework/operator-marketplace
+COPY . .
+RUN make build
+
+FROM centos:7
+RUN useradd marketplace-operator
+USER marketplace-operator
+COPY --from=builder /go/src/github.com/operator-framework/operator-marketplace/tmp/_output/bin/marketplace-operator /usr/bin
+
+LABEL io.k8s.display-name="OpenShift marketplace-operator" \
```

Comment:
For display, `OpenShift Marketplace Operator` maybe?

Reply:
+1